src/Approvals/SFA.DAS.Approvals/Application/Cohorts/Queries/GetCohortDetails/GetCohortDetailsQuery.cs

```diff
@@ -21,34 +27,54 @@ public class GetCohortDetailsQueryResult
     {
         public string ProviderName { get; set; }
         public string LegalEntityName { get; set; }
+        public bool HasUnavailableFlexiJobAgencyDeliveryModel { get; set; }
     }
 
     public class GetCohortDetailsQueryHandler : IRequestHandler<GetCohortDetailsQuery, GetCohortDetailsQueryResult>
     {
+        private readonly IDeliveryModelService _deliveryModelService;
```

Comment:
This doesn't appear to be used, other than in the ctor

Reply:
Agreed - removed, thanks